gem/Gemfile

```diff
@@ -16,8 +16,3 @@
 source 'https://rubygems.org'
 
 gemspec
-
-gem 'coveralls', require: false
-gem 'rspec'
-gem 'rubocop'
-gem 'thor' # enable: bundle exec thor spec
```

Comment:
I am getting an error:
```
Could not find command "spec".
```
I think your env is different than mine. what does your Gemfile.lock look like? you error seem to be you have missing gems, I assume you tried `bundle install` one more time?